pkgs/development/python-modules/treeo/default.nix

```diff
@@ -8,21 +8,16 @@
 
 buildPythonPackage rec {
   pname = "treeo";
-  version = "0.4.0";
+  version = "0.0.9";
```

Comment:
We are intentionally downgrading?

Reply:
Upgrading treeo, end up involving pulling on a thread that required updating a few other packages. I just force-pushed a new sequence of commits with all of the new updates.